build-system/test-configs/dep-check-config.js

```diff
@@ -192,12 +192,6 @@ exports.rules = [
       'extensions/amp-stream-gallery/0.1/amp-stream-gallery.js->extensions/amp-base-carousel/0.1/child-layout-manager.js',
       'extensions/amp-stream-gallery/0.1/amp-stream-gallery.js->extensions/amp-base-carousel/0.1/responsive-attributes.js',
 
-      // <amp-date-display> versions share these date format helpers
```

Comment:
q: do we plan to continue constraining cross-component dependencies between the components in the `src/bento/components` repo? this file looks like it only lists components in the `extensions` directory, but eventually the paths would be updated write?

Reply:
Yes I intend to update the paths once all the components are moved. Trying to update this with each batch of components seems like it would create a lot of duplicate work.